docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 
-<br/>
+To implement payments correctly, include a [`capacityDelegationAuthSig`](https://developer.litprotocol.com/sdk/capacity-credits#createcapacitydelegationauthsig) within the `capabilityAuthSigs` array when generating session signatures.
 
-Here is what each field means:
+## Storing `SessionSigs`
 
-- `sig` is the signature produced by the ed25519 keypair signing the `signedMessage` payload
-- `derivedVia` should be `litSessionSignViaNacl` and specifies that the SessionSig object was created via the `NaCl` library.
-- `signedMessage` is the payload that was signed by the session keypair.
-- `address` is the session keypair public key.
-- `algo` is the signing algorithm used to generate the session signature.
+When running code to generate session signatures, you can store the session signatures and session keys using `LocalStorage` from the `node-localstorage` package.
 
-### Signed Message
+```javascript
+import { LocalStorage } from "node-localstorage";
+```
 
-Here is what each field in `signedMessage` means:
+When running code within a browser, this import is not needed, as the session signatures and keys will be stored within the browser's local storage. However, when running this code in an environment such as Node.js where browser local storage is not available, the `LocalStorage` module can be used to provide file-based storage. 
 
-- `sessionKey` is the session keypair public key.
-- `resourceAbilityRequests` is a lit of abilities that the session key is requesting to perform against the specified Lit resources during authentication. Read more [here](resources-and-abilities) about Lit Resources and Abilities.
-- `capabilities` is an array of one or more AuthSigs.
-- `issuedAt` is the time the SessionSig was issued.
-- `expiration` is the time the SessionSig becomes invalid.
-- `nodeAddress` is the specific URL the SessionSig is meant for.
+```javascript
+litNodeClient = new LitNodeClient({
+    litNetwork: LitNetwork.DatilDev,
+    // This storageProvider object can be omitted if executing in a browser
+    storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+    },
+});
+```
+When a `storageProvider` is added to the `LitNodeClient` instance, all functions for generating session signatures will try to create a session keypair for you and store it in the local storage. 
 
-#### Capabilities
+The session keypair can also be generated with the `generateSessionKeyPair()` function. Doing this enables you to pass the generated session keypair as the optional `sessionKey` parameter when generating session signatures.
```

Comment:
This seems a little out of place for the storage section?

Reply:
How about this below the local storage section?:

> #### Manually Generating Session Keys
> The session keypair can also be generated with the `generateSessionKeyPair()` function. Doing this enables you to pass the generated session keypair as the optional `sessionKey` parameter when generating Session Signatures.